newIDE/app/src/ResourcesList/index.js

```diff
@@ -79,6 +83,38 @@ export default class ResourcesList extends React.Component<Props, State> {
     this.props.onDeleteResource(resource);
   };
 
+  _scanForNewResources = () => {
+    const resourcesList = this
+    const project = this.props.project
+    const resourcesManager = project.getResourcesManager()
+    console.log('Scanning the project folder for new resources...');
+    const projectPath = path.dirname(project.getProjectFile());
+
+    const getDirectories = (src, callback) => {
+      glob(src + '/**/*.png', callback);
+    };
+    getDirectories(projectPath, function (err, res) {
+      if (err) {
+        console.log('Error loading ', err);
+      } else {
+        res.forEach((pathFound)=>{
+          // should use relative path for resource name here!
+          let fileName = String(pathFound).split('/').pop();
```

Comment:
nitpicking: you can use const instead of let.

Also, use the functions provided `path` to manipulate filenames. You might be interested in `path.basename` (https://nodejs.org/api/path.html#path_path_basename_path_ext).

Reply:
ah you are right, it's probably better for multiplatform support too

Reply:
@4ian if we want to keep the slashes in the name - instead of path.basename  I could just use the relative path to name it?
It will help us avoid confusion

Reply:
@blurymind Yes use path.relative for the name, sounds like the most robust version.